pandas/errors/__init__.py

```diff
@@ -288,3 +288,23 @@ class SettingWithCopyError(ValueError):
     >>> df.loc[0:3]['A'] = 'a' # doctest: +SKIP
     ... # SettingWithCopyError: A value is trying to be set on a copy of a...
     """
+
+
+class SettingWithCopyWarning(Warning):
+    """
+    Warning is raised when trying to set on a copied slice from a dataframe and
+    the mode.chained_assignment is set to 'warn.' 'Warn' is the default option.
+    This can happen unintentionally when chained indexing.
+
+    For more information, see 'Evaluation order matters' on
+    https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html
```

Comment:
Could you use sphinx referencing here like

``` For more information, see :ref:`the user guide<indexing.evaluation_order>` ```

You can check `indexing.rst`  for the `.. _indexing.evaluation_order` sphinx references for each section.

Reply:
Ah, I didn't know about that; thank you for the comment! I've updated the references